File: backend/components/metiers/threed/legacy/fallback/samples/sample_fallback_legacy.js
// sample_fallback_legacy.js – Exemple ultra avancé fallback legacy
function sampleFallbackLegacy(data) {
  if (!data) return { fallback: true, empty: true };
  return { ...data, fallback: true };
}
module.exports = { sampleFallbackLegacy };

File: backend/components/metiers/administration_publique/legacy/fallback/samples/sample_fallback_legacy.py
# sample_fallback_legacy.py – Exemple ultra avancé fallback legacy
def sample_fallback_legacy(data):
    if not data:
        return {'fallback': True, 'empty': True}
    result = data.copy() if isinstance(data, dict) else {'value': data}
    result['fallback'] = True
    return result
